example/spec/parallel/createReconciliationReport/CreateReconciliationReportSpec.js

```diff
@@ -481,10 +480,11 @@ describe('When there are granule differences and granule reconciliation is run',
       expect(report.status).toBe('SUCCESS');
     });
 
-    it('generates a report showing cumulus files that are in S3 but not in the Cumulus files table', () => {
+    it('generates a filtered report, omitting files that are in S3 but not in the Cumulus files table', () => {
       if (beforeAllFailed) fail(beforeAllFailed);
       const extraS3ObjectUri = buildS3Uri(extraS3Object.Bucket, extraS3Object.Key);
-      expect(report.filesInCumulus.onlyInS3).toContain(extraS3ObjectUri);
+      expect(report.filesInCumulus.onlyInS3).not.toContain(extraS3ObjectUri);
```

Comment:
It was because before this change we did expect to find this object, so rather than just delete the expectation, I left is as no there and added the length zero.  I was just being overly explicit but don't think it hurts anything.